### DISCUSSION TITLE
Can't download pre-built image from Cloudsmith

### DISCUSSION THREAD

**Discussion by WilfulGrey** [bareboat-necessities/lysmarine_gen#428] [General]
Hi, I was trying yesterday, and I was trying today - every time I'm getting an error 402. Am I missing something?

**Comment by mgrouch** [comment]
You need to download using the script given in release notes. 

Unfortunately free cloudsmith service is capped by a limit on monthly downloads and it exceeded that limit for this month. 

Limit resets on 2nd day of each month.

**Reply by WilfulGrey** [reply]
Thanks @mgrouch !
